### PR TITLE
Test/test ci 2

### PR DESCRIPTION
**Pull request by opetliak**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -26,6 +26,14 @@ jobs:
         uses: docker/build-push-action@v4
         with:
           context: .
-          file: ./Dockerfile
+          file: ./docker/web/Dockerfile
+          push: true
+          tags: ${{ secrets.DOCKERHUB_USERNAME }}/web_image:latest
```
Comment (truskovskiyk) [inline .github/workflows/main.yml:31]: should we use something else then latest?

File: docker/training/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM python:3.8.2-slim-buster
+RUN useradd --create-home appuser
+USER appuser
+
+WORKDIR /home/appuser
+COPY . .
+RUN which python
+RUN python -m pip install --upgrade pip setuptools wheel
```
Comment (truskovskiyk) [inline docker/training/Dockerfile]: please install deps before copying all code

File: docker/training/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+ENV PYTHONFAULTHANDLER=1
```
Comment (truskovskiyk) [inline docker/training/Dockerfile:8]: why do we need this?

**Review by truskovskiyk — APPROVED** [review]
@opetliak amazing work, please fix comments and merge

File: docker/training/Dockerfile (hunk 3 above)
Comment (opetliak) [reply docker/training/Dockerfile:8]: what exactly?
PYTHONFAULTHANDLER?

File: docker/training/Dockerfile (hunk 2 above)
Comment (opetliak) [reply docker/training/Dockerfile]: done

**Pull request merged**